### DISCUSSION TITLE
get_antizapret_domains.sh + autohostlist

### DISCUSSION THREAD

**Discussion by synmais** [bol-van/zapret#345] [General]
Добрый день!

Прежде всего, огромное спасибо за проделанную работу!

Изначально хотел обойти замедление Youtube, что успешно удалось, бонусом заработали все остальные заблокированные сайты. Сначала использовал режим фильтрации "none", все шло очень неплохо, но потом все-таки стали появляться проблемы в отдельных приложениях типа NUM. Потом попробовал "autohostlist" - режим хорош, отрабатывает здорово, но первая загрузка заблокированного контента происходит либо с большой задержкой, либо со второго раза. Затем добавил вручную список antizapret и стало вообще здорово: так как большинство заблокированных сайтов есть в списке, то грузятся мгновенно и с первого раза, а те, которых нет, добавляются автоматически и после добавления работают так же хорошо.
Однако, в режиме фильтрации "hostlist" есть фишка, что список antizapret выкачивается автоматически в zapret-hosts.txt.gz и время от времени обновляется (?). 
Собственно вопрос. Возможно ли как-то скомбинировать использование и автоматическое обновление zapret-hosts.txt.gz и autohostlist?

С уважением, Павел

**Comment by bol-van** [comment]
инсталятор такое не предложит, но если в нем выбрать hostlist
а потом остановить zapret
в конфиге поправить
MODE_FILTER=autohostlist
и запустить снова
должно сработать

**Reply by synmais** [reply]
Благодарю! Получилось
